proposals/lip-0046.md

```diff
@@ -54,7 +54,7 @@ We define the following constants:
 | `ADDRESS_LENGTH`               | uint32   | 20              | Length in bytes of type Address.                |
 | `SEED_LENGTH`                  | uint32   | 16              | Length in bytes of a valid seed revealed.       |
 | **Config parameters**          |          | Mainchain value |                                                 |
-| `MAX_LENGTH_VALIDATOR_REVEALS` | uint32   | 206             | Maximum length of the `validatorReveals` array. |
+| `MAX_LENGTH_VALIDATOR_REVEALS` | uint32   | 206             | Maximum length of the `validatorReveals` array. It should be set as twice the maximum length of the chains validator set. |
```

Comment:
Good point, I think it means twice the length of a round, so that if a validator committed in the previous round we can always find their commited value (extreme case that in the previous round generated the first block and in the current round generates the last one).

Reply:
In general, it is double the round length.

Reply:
To make it clear, I rephrase it.

Reply:
I would say that this constant must be at least twice the length of the round for the Random module logic to work correctly.

Reply:
@sergeyshemyakov I have already updated it. See here https://github.com/LiskHQ/lips/pull/266#discussion_r1116683969